x-pack/plugins/cases/public/components/custom_fields/builder.tsx

```diff
@@ -10,7 +10,7 @@ import { CustomFieldTypes } from '../../../common/types/domain';
 import { configureTextCustomFieldFactory } from './text/configure_text_field';
 import { configureToggleCustomFieldFactory } from './toggle/configure_toggle_field';
 
-export const builderMap: CustomFieldBuilderMap = Object.freeze({
+export const builderMap = Object.freeze({
   [CustomFieldTypes.TEXT]: configureTextCustomFieldFactory,
   [CustomFieldTypes.TOGGLE]: configureToggleCustomFieldFactory,
-} as const);
+} as const) as CustomFieldBuilderMap;
```

Comment:
What is the difference?

Reply:
I made the types to accept template literals so we do not have to cast inside the View/Configure/Edit components. Without this cast, TS complains. I fought a lot of hours without success. I preferred to have this cast here and not have to cast in the components each time we add a new custom field type.